src/plugins/ui_stateful/sheetview.ts

```diff
@@ -296,6 +300,23 @@ export class SheetViewPlugin extends UIPlugin {
     return Math.max(...this.getSubViewports(sheetId).map((viewport) => viewport.getColIndex(x)));
   }
 
+  /**
+   * Return the index of a column given an offset x, based on the viewport left
+   * visible cell.
+   * If the position is outside of any viewport, return the column index on the main viewport.
+   */
+  getColIndexHidden(x: Pixel): HeaderIndex {
+    const colIndex = this.getColIndex(x);
```

Comment:
I have a hard time understanding the implementation :/
We should also find a clearer name imho